Description: Allow env var QTWEBKIT_DPR to define a default init value
Author: Christian Dywan <christian.dywan@canonical.com>
Origin: vendor
Forwarded: not-needed

--- wk-5.0.orig/Source/WebCore/page/Page.cpp
+++ wk-5.0/Source/WebCore/page/Page.cpp
@@ -143,7 +143,7 @@ Page::Page(PageClients& pageClients)
     , m_areMemoryCacheClientCallsEnabled(true)
     , m_mediaVolume(1)
     , m_pageScaleFactor(1)
-    , m_deviceScaleFactor(1)
+    , m_deviceScaleFactor(atof(getenv("QTWEBKIT_DPR")?getenv("QTWEBKIT_DPR"):"1.0"))
     , m_suppressScrollbarAnimations(false)
     , m_didLoadUserStyleSheet(false)
     , m_userStyleSheetModificationTime(0)
@@ -678,7 +678,7 @@ void Page::setDeviceScaleFactor(float sc
     if (m_deviceScaleFactor == scaleFactor)
         return;
 
-    m_deviceScaleFactor = scaleFactor;
+    m_deviceScaleFactor = atof(getenv("QTWEBKIT_DPR")?getenv("QTWEBKIT_DPR"):"1.0");
     setNeedsRecalcStyleInAllFrames();
 
 #if USE(ACCELERATED_COMPOSITING)
